Makefile

```diff
@@ -0,0 +1,10 @@
+DESTDIR ?=
+PREFIX ?= $(DESTDIR)/usr
```

Comment:
Overriding both DESTDIR and PREFIX will produce a surprising result.

Reply:
Fixed, thanks.